ehrapy/tools/feature_ranking/_rank_features_groups.py

```diff
@@ -240,6 +244,51 @@ def _evaluate_categorical_features(
     )
 
 
+def _check_no_datetime_columns(df):
+    datetime_cols = [col for col in df.columns if df[col].dtype == "datetime64[ns]"]
```

Comment:
```suggestion
datetime_cols = [col for col in df.columns if pd.api.types.is_datetime64_any_dtype(df[col]) or pd.api.types.is_timedelta64_dtype(df[col])]
```

Just an idea. Not sure whether it's needed or better.